execution_engine_testing/tests/src/test/contract_messages.rs

```diff
@@ -579,6 +593,18 @@ fn should_carry_message_topics_on_upgraded_contract() {
     assert_eq!(expected_topic_names, 2);
 }
 
+#[ignore]
+#[test]
+fn should_carry_message_topics_on_upgraded_contract_without_initializer() {
```

Comment:
```suggestion
fn should_carry_message_topics_on_upgraded_contract_with_initializer() {
```

Reply:
Renamed in: https://github.com/casper-network/casper-node/pull/4461/commits/08f12c8722a313d044bfe397297e95a695cb53e2